src/mpid/ch4/netmod/ofi/subconfigure.m4

```diff
@@ -211,6 +210,7 @@ AM_COND_IF([BUILD_CH4_NETMOD_OFI],[
         if test "x$ofi_direct_provider" != "x" ; then
             PAC_APPEND_FLAG([-I${master_top_builddir}/src/mpid/ch4/netmod/ofi/libfabric/prov/${ofi_direct_provider}/include], [CPPFLAGS])
             PAC_APPEND_FLAG([-I${use_top_srcdir}/src/mpid/ch4/netmod/ofi/libfabric/prov/${ofi_direct_provider}/include], [CPPFLAGS])
+            PAC_APPEND_FLAG([-DFABRIC_DIRECT],[CPPFLAGS])
```

Comment:
I'm not understanding what this flag is for. The commit message says this is for building MPICH, but `git grep FABRIC_DIRECT` does not show any instances of it being used.